includes/reports/overview/list-table/class-sensei-reports-overview-list-table-students.php

```diff
@@ -149,25 +149,14 @@ protected function get_row_data( $item ) {
 		$user_email = $item->user_email;
 
 		// Output the users data.
-		if ( $this->csv_output ) {
-			$user_name = Sensei_Learner::get_full_name( $item->ID );
-		} else {
-			$url                 = add_query_arg(
-				array(
-					'page'      => $this->page_slug,
-					'user_id'   => $item->ID,
-					'post_type' => $this->post_type,
-				),
-				admin_url( 'edit.php' )
-			);
-			$user_name           = '<strong><a class="row-title" href="' . esc_url( $url ) . '">' . esc_html( $item->display_name ) . '</a></strong>';
+		if ( ! $this->csv_output ) {
 			$user_average_grade .= '%';
 		}
 
 		$column_data = apply_filters(
 			'sensei_analysis_overview_column_data',
 			array(
-				'title'             => $user_name,
+				'title'             => $this->format_user_name( $item->ID, $this->csv_output ),
```

Comment:
Hm, this looks like to be the opposite logic - right now if you do a CSV export, you see the links and there are no links on the admin page.

Reply:
Oh Gosh! I tested but apparently, I was seeing a cached version, sorry about that!
Fixed 446c042c18b310acae370bc007133baa55ba0ab1